eng/common/TestResources/New-TestResources.ps1

```diff
@@ -322,11 +328,14 @@ if ($TenantId) {
 if ($TestApplicationSecret) {
     $templateParameters.Add('testApplicationSecret', $TestApplicationSecret)
 }
-if ($AdditionalParameters) {
-    $templateParameters += $AdditionalParameters
+if ($ArmTemplateParameters) {
```

Comment:
We should also look and set the variables here just in case there are any parameters being overridden.  I would also like us to add some warning messages if we are overridding a value because that will most likely cause issues and it would be nice to be able to diagnose that if it happens.